sympy/stats/joint_rv.py

```diff
@@ -330,3 +351,12 @@ def marginalise_out(self, expr, rv):
 
     def __call__(self, *args):
         return self.pdf(*args)
+
+class StochasticProcess(JointDistribution):
+    """
+    Class representing stochastic processes.
+    """
+
+    @property
+    def component_count(self):
```

Comment:
We may have countable and uncountable stochastic processes. We need to consider that.